acstools/acsrej.py

```diff
@@ -215,6 +231,4 @@ def run(configobj=None):
            crradius=configobj['crradius'],
            crthresh=configobj['crthresh'],
            badinpdq=configobj['badinpdq'],
-           newbias=configobj['newbias'],
-           exe_args=configobj['exe_args']
```

Comment:
Note: `exe_args` is not a part of the TEAL interface. In fact, TEAL interface does not understand Python list.

If we want to support arbitrary `exe_args` in TEAL, it needs to be read in as one long comma-separated string and then parsed into a list. Such work is beyond the scope of this PR. Also argument could be made that such an option can be very confusing for the user on the GUI side.